components/pendingRewards.ts

```diff
@@ -53,7 +62,7 @@ const pendingRewardsCtrl = async (prisma: PrismaClient, body: string) => {
         })
 
         let epochRatio = delta/432000000
-        let updatePay = Math.floor(epochRatio*(poolargs.poolInfo.rewardPerEpochQt))
+        let updatePay = Math.floor(epochRatio * ( quantity * (poolargs.poolInfo.rewardPerEpochQt)))
```

Comment:
If the pool has more than 1 token we want to have the ability that each token has its own rewardPerEpochQt. (Example: token 1 has 500 rewardPerEpochQt, token 2 has 300 rewardPerEpochQt etc. ...

Reply:
@marjankhz I think it will be very helpful to test the full application. Then you can also verify if your solution works.